### PR TITLE
Add Angha50 test to CI

### PR DESCRIPTION
**Pull request by tetsuo-cpp**

### CODE REVIEW

**Comment by tetsuo-cpp** [comment]
Don't merge until https://github.com/lifting-bits/lifting-tools-ci/pull/16 goes in.

**Comment by tetsuo-cpp** [comment]
Cool! This exposed a few failures. I'll make some fixes and get this passing.

**Comment by tetsuo-cpp** [comment]
One failure is solved by: https://github.com/lifting-bits/anvill/pull/393

The other one is a bit odd and I can't seem to repro it locally.
```
ERROR REPORT SCRIPT ERROR:  ( /__w/anvill/anvill/src/anvill/angha-test-50/binaries/arm64/linux/drivers/net/ethernet/intel/i40e/ext
r_i40e_common.c_i40e_aq_get_link_info.o )  /__w/anvill/anvill/src/irene3/deps/ghidra/Ghidra/Extensions/irene-ghidra/ghidra_scripts
/anvillHeadlessExportScript.java : Cannot invoke "ghidra.program.model.lang.Register.getBaseRegister()" because "reg" is null (Hea
dlessAnalyzer) java.lang.NullPointerException: Cannot invoke "ghidra.program.model.lang.Register.getBaseRegister()" because "reg"
is null
        at anvill.Util$ProgramAnalysisUtilMixin.registerToDefinedVnode(Util.scala:59)
        at anvill.Util$ProgramAnalysisUtilMixin.vnodeToBasRegVnodeOrUnique$$anonfun$2$$anonfun$1(Util.scala:68)
        at scala.Option$.when(Option.scala:41)
        at anvill.Util$ProgramAnalysisUtilMixin.vnodeToBasRegVnodeOrUnique$$anonfun$2(Util.scala:68)
        at scala.Option.orElse(Option.scala:477)
        at anvill.Util$ProgramAnalysisUtilMixin.vnodeToBasRegVnodeOrUnique(Util.scala:69)
        at anvill.StackPointsTo.assignIntoReg(StackPointsTo.scala:304)
        at anvill.StackPointsTo.$anonfun$5(StackPointsTo.scala:330)
        at scala.Option.map(Option.scala:242)
        at anvill.StackPointsTo.update_op(StackPointsTo.scala:330)
        at anvill.StackPointsTo.update_op(StackPointsTo.scala:311)
        at anvill.ComputeNodeContext$package$given_Problem_D_CfgEdge.update_edge(ComputeNodeContext.scala:197)
        at anvill.ComputeNodeContext$package$given_Problem_D_CfgEdge.update_edge(ComputeNodeContext.scala:192)
        at anvill.Fixpoint$.fixpoint$$anonfun$1(ComputeNodeContext.scala:145)
        at scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
        at scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
        at scala.collection.immutable.Set$Set1.foreach(Set.scala:168)
        at anvill.Fixpoint$.fixpoint(ComputeNodeContext.scala:173)
        at anvill.ComputeNodeContext$.fixpoint(ComputeNodeContext.scala:424)
        at anvill.ComputeNodeContext$.node_fixpoint(ComputeNodeContext.scala:432)
        at anvill.TypeAnalysis.analyzePointsTo(TypeAnalysis.scala:52)
        at anvill.TypeAnalysis.analyzeWithAddressSetView(TypeAnalysis.scala:78)
        at anvill.TypeAnalysis.analyze(TypeAnalysis.scala:99)
        at anvill.ProgramSpecifier$.computeTypeHints(ProgramSpecifier.scala:617)
        at anvill.ProgramSpecifier$.specifyFunction(ProgramSpecifier.scala:668)
        at anvill.ProgramSpecifier$.specifyFunctionOrDecl(ProgramSpecifier.scala:975)
        at anvill.ProgramSpecifier$.$anonfun$24(ProgramSpecifier.scala:1173)
        at scala.collection.immutable.List.map(List.scala:250)
        at scala.collection.immutable.List.map(List.scala:79)
        at anvill.ProgramSpecifier$.specifyProgram(ProgramSpecifier.scala:1174)
        at anvill.ProgramSpecifier$.specifyProgram(ProgramSpecifier.scala:1326)
        at anvill.ProgramSpecifier.specifyProgram(ProgramSpecifier.scala)
        at anvillHeadlessExportScript.run(anvillHeadlessExportScript.java:22)
        at ghidra.app.script.GhidraScript.executeNormal(GhidraScript.java:397)
        at ghidra.app.script.GhidraScript.doExecute(GhidraScript.java:252)
        at ghidra.app.script.GhidraScript.execute(GhidraScript.java:230)
        at ghidra.app.util.headless.HeadlessAnalyzer.runScript(HeadlessAnalyzer.java:579)
        at ghidra.app.util.headless.HeadlessAnalyzer.runScriptsList(HeadlessAnalyzer.java:912)
        at ghidra.app.util.headless.HeadlessAnalyzer.analyzeProgram(HeadlessAnalyzer.java:1060)
        at ghidra.app.util.headless.HeadlessAnalyzer.processFileWithImport(HeadlessAnalyzer.java:1544)
        at ghidra.app.util.headless.HeadlessAnalyzer.processWithImport(HeadlessAnalyzer.java:1662)
        at ghidra.app.util.headless.HeadlessAnalyzer.processWithImport(HeadlessAnalyzer.java:1727)
        at ghidra.app.util.headless.HeadlessAnalyzer.processLocal(HeadlessAnalyzer.java:448)
        at ghidra.app.util.headless.AnalyzeHeadless.launch(AnalyzeHeadless.java:127)
        at ghidra.GhidraLauncher.launch(GhidraLauncher.java:77)
        at ghidra.Ghidra.main(Ghidra.java:54)

INFO  ANALYZING changes made by post scripts: /__w/anvill/anvill/src/anvill/angha-test-50/binaries/arm64/linux/drivers/net/ethernet/intel/i40e/extr_i40e_common.c_i40e_aq_get_link_info.o (HeadlessAnalyzer)
INFO  REPORT: Post-analysis succeeded for file: /__w/anvill/anvill/src/anvill/angha-test-50/binaries/arm64/linux/drivers/net/ethernet/intel/i40e/extr_i40e_common.c_i40e_aq_get_link_info.o (HeadlessAnalyzer)
INFO  REPORT: Discarded file import due to readOnly option: /extr_i40e_common.c_i40e_aq_get_link_info.o (HeadlessAnalyzer)
INFO  REPORT: Import succeeded (HeadlessAnalyzer)
```

File: .github/workflows/build.yml
Comment (tetsuo-cpp) [inline .github/workflows/build.yml]: - [x] Revert this before merging.

File: lib/Lifters/ValueLifter.cpp
Comment (tetsuo-cpp) [inline lib/Lifters/ValueLifter.cpp]: - [x] Rebase and get rid of this before merging. This is in a separate PR, I'm just bringing it in for testing purposes.

File: ci/angha_50_test_settings.json
Comment (tetsuo-cpp) [inline ci/angha_50_test_settings.json]: A few tests were timing out. We really shouldn't need this since each program contains a single function as far as I can tell.

I wonder if this is related to the perf degradation that happened when that liveness refactor got merged in @2over12.

Comment (tetsuo-cpp) [reply ci/angha_50_test_settings.json]: Turns out I just had to use the `--jobs` flag I added to `lifting-tools-ci`. No problem.

**Comment by tetsuo-cpp** [comment]
@2over12 This should be ready to go now.

**Review by 2over12 — APPROVED** [review]
lgtm, thanks!

**Pull request merged**
